### PR TITLE
chore(build): remove some scripts in favor of `emit-module-package-file`

### PR DESCRIPTION
**Pull request by Andarist**
Just a small trick that colocates generating this file with the actual build that depends~ on it.

### CODE REVIEW

**Comment by pkerschbaum** [comment]
LGTM

File: rollup.config.js [hunk 1]
```diff
@@ -46,7 +46,20 @@ export default [
       file: 'dist/esm/tiny-invariant.js',
       format: 'esm',
     },
-    plugins: [typescript({ module: 'ESNext' })],
+    plugins: [
+      typescript({ module: 'ESNext' }),
+      // https://github.com/rollup/rollup/blob/69ff4181e701a0fe0026d0ba147f31bc86beffa8/build-plugins/emit-module-package-file.ts
+      {
+        generateBundle() {
```
Comment (pkerschbaum) [inline rollup.config.js:53]: would that call `generateESMPackageJson` or something instead of `generateBundle`?

Comment (Andarist) [reply rollup.config.js:53]: hm, this is just a plugin hook - it has to use a specific name to be called at the appropriate moment

Comment (pkerschbaum) [reply rollup.config.js:53]: Ah ok, didn't know that!

**Pull request merged**